software/contrib/cvecorder.py

```diff
@@ -236,6 +286,10 @@ def loadState(self):
         # For each bank, check if a state file exists, then load it
         # If not, initialize the bank with zeros
 
+        # Potential issue......
+        # If for some reason the file open command fails, it will init each bank and wipe any previous recordings
+        # Added retries and debug code to try and capture the error if this does occur
```

Comment:
[optional] This is a helpful troubleshooting comment, also consider adding a link to the github issue for further context.